src/main/java/org/interledger/cryptoconditions/ConditionImpl.java

```diff
@@ -2,17 +2,29 @@
 
 import java.util.EnumSet;
 
+
 import org.interledger.cryptoconditions.Condition;
 import org.interledger.cryptoconditions.ConditionType;
 import org.interledger.cryptoconditions.FeatureSuite;
+import org.interledger.cryptoconditions.encoding.Base64Url;
 
-final class ConditionImpl implements Condition {
+public final class ConditionImpl implements Condition {
```

Comment:
Again, this is forced by Java syntax. Without the public the ConditionInputStream doesn't compile. (previously ConditionInputStream was instantiating an anonymous Condition, but it was not a real implementation).